nwbwidgets/base.py

```diff
@@ -198,10 +198,11 @@ def show_text_fields(node, exclude=('comments', 'interval'), **kwargs):
     return widgets.VBox(info)
 
 
-def plot_traces(time_series: TimeSeries, time_start=0, time_duration=None, trace_window=None,
-                title: str = None, ylabel: str = 'traces'):
+def plot_traces(time_series: TimeSeries, time_start = 0, time_duration = None, trace_window = None,
+                n_trace = 1, title: str = None, ylabel: str = 'traces'):
```

Comment:
1) don't put spaces around `=` in function calls
2) n_traces should be computed from trace_window, not supplied in the function call

Reply:
For (2), when the `trace_window` is `None` , it needs to calculate `time_series.data.shape[1]`. We can make the `trace_window` mandatory in the function.

Reply:
you can use `len(time_series.data[0])` if the `shape` attribute is not available.

Reply:
Yes, but still calculating `mini_data` needs multi-axis slicing which may require for loop or other slicing operators when `time_series.data` is not a numpy array.